src/pixelator/analysis/polarization/__init__.py

```diff
@@ -8,10 +8,8 @@
 
 from pixelator.graph.utils import Graph, create_node_markers_counts
 from pixelator.statistics import (
-    binarize_counts,
```

Comment:
If in this PR, you are removing the binarize and denoise functionalities, you need to do that for the functions but also everything that refers to it in the tests as well.